ci/infra/azure/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 0.12"
+
+  # required_providers {
+  #   azurerm  = "~> 2.5"
+  #   template = "~> 2.1"
+  #   local    = "~> 1.2"
+  #   null     = "~> 2.0"
+  # }
```

Comment:
We should either uncomment that or delete it.

Reply:
removed